packages/veritone-widgets/package.json

```diff
@@ -25,7 +25,8 @@
     "redux-thunk": "^2.3.0",
     "shaka-player": "^2.4.1",
     "veritone-oauth-helpers": "^2.1.0",
-    "video-react": "~0.13.3"
+    "video-react": "~0.13.3",
+    "reselect": "^4.0.0"
```

Comment:
Why did we add reselect, remove it if not used.

Reply:
I saw it at https://github.com/veritone/veritone-sdk/blob/master/packages/veritone-widgets/src/redux/modules/folder/selector.js#L1